elements/pl-file-preview/pl-file-preview.py

```diff
@@ -34,14 +34,15 @@ def render(element_html, data):
         for idx, file in enumerate(submitted_files):
             b64contents = file["contents"] or ""
             try:
-                contents = base64.b64decode(b64contents).decode()
+                base64.b64decode(b64contents).decode()
+                decode_error = False
```

Comment:
Instead of using a boolean variable, you can just set `decode_error = "false"` and then assign it in the dictionary directly.